koptional-rxjava2-extensions/src/main/kotlin/com/gojuno/koptional/rxjava2/Flowable.kt

```diff
@@ -0,0 +1,28 @@
+package com.gojuno.koptional.rxjava2
+
+import com.gojuno.koptional.None
+import com.gojuno.koptional.Optional
+import com.gojuno.koptional.Some
+import io.reactivex.Flowable
+
+private inline fun <reified R : Any> Flowable<*>.ofType(): Flowable<R> = ofType(R::class.java)
+
+/**
+ * Filters items emitted by a [Flowable] by only emitting those that are [Some].
+ *
+ * Does not operate by default on a particular [Scheduler][io.reactivex.Scheduler].
```

Comment:
KDoc doesn’t support HTML tags, including images ([YouTrack](https://youtrack.jetbrains.com/issue/KT-30880)). Replaced them with a regular KDoc syntax. Fun fact — KDoc not only doesn’t support these but hides them completely. Meaning that the scheduler notice wasn’t in the documentation forever.